plugins/answer.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+from urllib.parse import quote, urljoin
+
+from errbot import BotPlugin, botcmd
+import requests
+
+
+class Answer(BotPlugin):
+
+    @botcmd
+    def answer(self, msg, arg):
```

Comment:
You should add error handling here. In case the plugin fails it should give a suitable message rather than no message. Or is that already there by default. Give it a thumbs down if I forget.